package.json

```diff
@@ -17,10 +17,7 @@
     "eslint-plugin-standard": "^2.1.1",
     "fixpack": "^2.3.1",
     "flow-bin": "^0.42.0",
-    "jest": "^19.0.2",
-    "remark-cli": "^3.0.0",
-    "remark-lint": "^6.0.0",
-    "remark-preset-lint-recommended": "^2.0.0"
```

Comment:
Well, it was picking up lots of non-errors and generating warnings. Well, I think they're non-errors. Like indentation and me using [1] for numbering. Personally I'd rather not have it than have to either comply with it's opinions or ignore it's warnings. But that's why I asked you to check this ;)